marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/mapping-functions/mapping-functions-sequence-parameter/test-data/entities/Customer.entity.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Please use the new createSimpleMappingProject helper in data-hub-test-helper.sjs. You can add the "items" array to the Customer entity model, though let's call it "integers" to make it clear that these are integers. 

The intent of this helper is to cut down on the number of flows/entities/steps that we have to create as separate files.